src/Nethermind/Nethermind.Network.Test/P2P/Subprotocols/Eth/V66/Eth66ProtocolHandlerTests.cs

```diff
@@ -61,8 +61,8 @@ public class Eth66ProtocolHandlerTests
         private IMessageSerializationService _svc;
         private ISyncServer _syncManager;
         private ITxPool _transactionPool;
-        private IPooledTxsRequestor _pooledTxsRequestor;
-        private IGossipPolicy gossipPolicy;
+        private IPooledTxsRequestor _pooledTxsRequester;
```

Comment:
Minor thing: I don't like this change, we discussed it in the past and decided to change form from requester to requestor. And form `requestor` is used in other parts of code. But it's cosmetic and only in tests so feel free to ignore